deforestation_api/dependencies/queryparams.py

```diff
@@ -23,7 +23,7 @@ def date_range(
     start_year: Annotated[
         int,
         Query(description="First year to return forest cover loss data for."),
-    ] = 2020,
+    ] = 2001,
```

Comment:
Is this because this is the earliest date we have data for?

Reply:
Yes, I thought that since there is only 21 time steps we might as well return the entire series by default.